pkg/aws/ec2/ec2.go

```diff
@@ -48,17 +49,25 @@ var (
 		[]string{"instance", "region", "family", "machine_type", "cluster_name", "price_tier"},
 		nil,
 	)
+	persistentVolumeHourlyCostDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(cloudcostexporter.MetricPrefix, subsystem, "persistent_volume_usd_per_hour"),
+		"The cost of an AWS EBS Volume in USD.",
+		[]string{"persistent_volume", "region", "availability_zone", "disk", "type", "size_gib", "state"},
+		nil,
+	)
```

Comment:
We don't have a way to automate this, when you have a moment can you add an entry to the [ec2 metrics documentation](https://github.com/grafana/cloudcost-exporter/blob/main/docs/metrics/aws/ec2.md)?

Reply:
done. Marking this thread as resolved, but happy to hear feedback on the documentation added itself too.